tests/src/python/test_provider_oracle.py

```diff
@@ -168,6 +168,20 @@ def testAddFeatureWrongGeomType(self):
         """
         pass
 
+    def testDeleteFeaturesUpdateExtent(self):
+        # FIXME
+        """
+        This test does not pass for the Oracle provider.
```

Comment:
These ones pass for me -- but again, I suspect the deeper underlying issue is causing instability here.